README.md

```diff
@@ -41,3 +41,17 @@ Otherwise, [bug reports](https://github.com/umbraco/UmbracoDocs/issues/), [bug f
 Look at the [Contributor Guidelines](CONTRIBUTING.md) to learn how you can get involved and help with the Umbraco Documentation.
 ## License [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE.md)
 This library is released under the [MIT License](LICENSE.md).
+
+# Previewing rendered output locally
+
+There is an experimental project which supports rendering the documentation locally.
+
+You can install this from [nuget](https://www.nuget.org/packages/Umbraco.Docs.Preview.App/) as a dotnet global tool by running the following in a terminal.
+
+```bash
+$ dotnet tool install -g Umbraco.Docs.Preview.App
+```
+
+Or check out the source at https://github.com/umbraco/UmbracoDocs.Preview
+
+Once installed, navigate to your local clone of the UmbracoDocs repository in a terminal and run the command `umbracodocs`
```

Comment:
```suggestion
Once installed, navigate to your local clone of the UmbracoDocs repository and run the command `umbracodocs`. This will start a local webserver (the URL will be shown), which you can open in your browser and view the rendered documentation.
```